cdc-accelerator/feature/smsTemplates.test.js

```diff
@@ -0,0 +1,62 @@
+import axios from 'axios'
+import SmsTemplates from './smsTemplates.js'
+import { smsExpectedResponse, getSmsExpectedResponseWithMinimumTemplates } from './test.gigyaResponse.sms.js'
+import { expectedGigyaResponseNok, getSiteConfig } from './test.gigyaResponses.js'
+import fs from 'fs'
+import path from 'path'
+import { credentials, apiKey, srcSiteDirectory } from './test.common.js'
+
+jest.mock('fs')
+jest.mock('axios')
+
+describe('Sms templates test suite', () => {
+    let smsTemplates
+
+    beforeEach(() => {
+        jest.clearAllMocks()
+        smsTemplates = new SmsTemplates(credentials)
+        jest.spyOn(smsTemplates, 'generateOtpTemplateFiles').mockImplementation(() => {})
```

Comment:
We can't mock the implementation of methods that are an important part of the feature being tested.
This way the feature is not being tested at all. How can we make sure the right files are created with the right content?